Module#02/.github/workflows/maven.yml

```diff
@@ -0,0 +1,26 @@
+name: Java CI with Maven(Gift-shop)
+
+on:
+  push:
+    branches: [ module#2 ]
+  pull_request:
+    branches: [ module#2 ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
```

Comment:
here java 11 version is mentioned but in maven-tomcat.yml java 18 is used. all configuration should be consistent.